config/locales/ja.yml

```
@@ -13,3 +14,4 @@ ja:
     errors:
       messages:
         blank: を入力してください
+        too_long: が長すぎます
```

Idiomatic

何文字以内にしないといけないかを書いてあげたほうが、ユーザーにとって使いやすいと思うので、
以下のように項目名と文字数も表示するようにしよう。
```suggestion
        too_long: "%{attribute}は%{count}文字以内で入力してください。"
```

---

たしかに
attributeは上で指定してるから%{count}を追加しました。
```
 attributes:
   user:
     name: アカウント名
```
[267f050](https://github.com/ok-os-job-change-team/kaito-twitter-clone-bootcamp/pull/87/commits/267f0505835a1d9bee1cc92c498d81594c922638)